src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,7 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+
+    public String getPrintableString();
+
+}
```

Comment:
Missing header comment. All non trivial methods and classes should have javadoc format header comments.